src/app/hooks/useClickTrackerHandler/index.jsx

```diff
@@ -99,9 +101,10 @@ const useClickTrackerHandler = (eventTrackingData = {}) => {
               useReverb,
               ...(groupTracker && { groupTracker }),
               ...(itemTracker && { itemTracker }),
-              ...(optimizelyVariation &&
-                optimizelyVariation !== 'off' && {
-                  experimentVariant: optimizelyVariation,
+              ...(experimentVariation &&
+                experimentVariation !== 'off' && {
+                  experimentName,
+                  experimentVariant: experimentVariation,
```

Comment:
Any chance we could rename the prop (perhaps at the source, or in a follow up PR?), so that we don't ned to do this
```suggestion
                  experimentVariant,
```